index.js

```diff
@@ -4,7 +4,7 @@
 // License text available at https://opensource.org/licenses/MIT
 
 var SG = require('strong-globalize');
-SG.SetRootDir(__dirname, {autonomousMsgLoading: 'all'});
+SG.SetRootDir(__dirname, { autonomousMsgLoading: 'all' });
```

Comment:
yeah that space was actually added when I ran a linting test. It is still at `1.0.0`.
I will fix that once I am done with this PR.